crates/teloxide/src/dispatching/dialogue.rs

```diff
@@ -93,23 +93,23 @@
 //!
 //! [`examples/dialogue.rs`]: https://github.com/teloxide/teloxide/blob/master/examples/dialogue.rs
 
-#[cfg(feature = "redis-storage")]
-pub use crate::dispatching::dialogue::{RedisStorage, RedisStorageError};
+/// This module was moved to [`teloxide::storage`].
+///
+/// [`teloxide::storage`]: crate::storage
+#[deprecated]
+pub mod storage {
+    pub use crate::storage::*;
+}
 
-#[cfg(feature = "sqlite-storage")]
-pub use crate::dispatching::dialogue::{SqliteStorage, SqliteStorageError};
+pub use crate::storage::*;
+pub use get_chat_id::GetChatId;
 
+use crate::{dispatching::DpHandlerDescription, types::ChatId};
 use dptree::{prelude::DependencyMap, Handler};
-pub use get_chat_id::GetChatId;
-pub use storage::*;
-use teloxide_core::types::ChatId;
 
 use std::{fmt::Debug, marker::PhantomData, sync::Arc};
 
-use super::DpHandlerDescription;
-
 mod get_chat_id;
-mod storage;
```

Comment:
Actually... I think this was a mistake. We say that the storage API is specifically for dialogues and this is just a private module. Having storages in `dispatching::dialogue` doesn't sound too bad (again, since the storage is said to be specifically for dialogues).

@Hirrolot what do you think? Should I revert this change?

Reply:
Go for it.

Reply:
Done